test/mkmf/base.rb

```diff
@@ -147,7 +147,7 @@ def config_value(name)
 
   include Base
 
-  def assert_separately(args, src, *rest, **options)
-    super(args + ["-r#{__FILE__}"], "extend TestMkmf::Base; setup\nEND{teardown}\n#{src}", *rest, **options)
+  def assert_separately(args, extra_args, src, *rest, **options)
+    super(args + ["-r#{__FILE__}"] + extra_args, "extend TestMkmf::Base; setup\nEND{teardown}\n#{src}", *rest, **options)
```

Comment:
We could always add `- --` here.

Or, instead of adding `extra_args`, what about move `args` after `"-r#{__FILE__}"`?

Reply:
> We could always add - -- here.

I modified on your suggestion above.